eth/api_miner.go

```diff
@@ -80,7 +64,7 @@ func (api *MinerAPI) SetEtherbase(etherbase common.Address) bool {
 	return true
 }
```

Comment:
This should probably be nuked (SetEtherbase) with all it's components across all packages.